### PR TITLE
Update collect-investigation-package.md

### PR DESCRIPTION
**Pull request by msbemba**
updated response when there is a collection that is already running.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5900

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba - minor edit, please.

**Comment by msbemba** [comment]
@JohanFreelancer9 I dont see any changes.

File: microsoft-365/security/defender-endpoint/collect-investigation-package.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ Comment|String|Comment to associate with the action. **Required**.
 
 ## Response
 
-If successful, this method returns 201 - Created response code and [Machine Action](machineaction.md) in the response body.
+If successful, this method returns 201 - Created response code and [Machine Action](machineaction.md) in the response body. If a collection is already running this returns 400 Bad Request.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/collect-investigation-package.md]:
```suggestion
If successful, this method returns 201 - Created response code and [Machine Action](machineaction.md) in the response body. If a collection is already running, this returns 400 Bad Request.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @msbemba, please just insert the comma. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**